packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -458,6 +539,71 @@ final TextTreeConfiguration transitionTextConfiguration = TextTreeConfiguration(
   isBlankLineBetweenPropertiesAndChildren: false,
 );
 
+/// Configuration that draws a box around a node ignoring the connection to the
+/// parents.
+///
+/// If nested in a tree, this node is best displayed in the property box rather
+/// than as a traditional child.
+///
+/// Used to draw a decorative box around detailed descriptions of an exception.
+///
+/// Example:
+/// ```
+/// ══╡ <name>: <description> ╞═════════════════════════════════════
+/// <body>
+/// ...
+/// ├─<normal_child_name>: <child_description>
+/// ╎ │ <property1>
+/// ╎ │ <property2>
+/// ╎ │ ...
+/// ╎ │ <propertyN>
+/// ╎ │
+/// ╎ └─<child_name>: <child_description>
+/// ╎     <property1>
+/// ╎     <property2>
+/// ╎     ...
+/// ╎     <propertyN>
+/// ╎
+/// ╎╌<dashed_child_name>: <child_description>
+/// ╎ │ <property1>
+/// ╎ │ <property2>
+/// ╎ │ ...
+/// ╎ │ <propertyN>
+/// ╎ │
+/// ╎ └─<child_name>: <child_description>
+/// ╎     <property1>
+/// ╎     <property2>
+/// ╎     ...
+/// ╎     <propertyN>
+/// ╎
+/// └╌<dashed_child_name>: <child_description>'
+/// ════════════════════════════════════════════════════════════════
+/// ```
+///
+/// /// See also:
+///
+///  * [DiagnosticsTreeStyle.error]
```

Comment:
Can you add one line describing why I should look there?

Reply:
done.